src/components/SearchForm/SearchForm.js

```diff
@@ -1,13 +1,22 @@
-import React, { useState } from 'react';
-import NothingFound from '../NothingFound/NothingFound';
-import Preloader from '../Preloader/Preloader';
+import React from 'react';
 
 function SearchForm(props) {
 
+    function onSubmit(e) {
+        e.preventDefault();
+        props.onSubmit(props.search);
+    }
+
     return (
         <form className='search-form'>
-            <input className='search-form__bar' type='text' id='search' placeholder='Enter topic' onChange={props.searchChange} />
-            <button className='button search-form__submit' type='submit' value='Search' onClick={props.onSubmit}>Search</button>
+            <input
+                className='search-form__bar'
+                type='text'
+                id='search'
+                placeholder='Enter topic'
+                onChange={props.onChange}
+                value={props.search} />
+            <button className='button search-form__submit' type='submit' value='Search' onClick={onSubmit}>Search</button>
```

Comment:
`onClick={onSubmit}`

* Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses